data/transactions/logic/evalStateful_test.go

```diff
@@ -678,9 +678,8 @@ pop
 		})
 	}
 
-	// check ed25519verify and arg are not allowed in statefull mode
+	// check that arg is not allowed in stateful mode
 	disallowed := []string{
-		"byte 0x01\nbyte 0x01\nbyte 0x01\ned25519verify",
```

Comment:
this should be checked for TEAL v2-v4.

Reply:
Will add a separate test that checks `verify` is not allowed in v4 and lower, and add another check that checks if `arg` is not allowed in v5 and beyond.